### PR TITLE
tests: update tests

### PR DESCRIPTION
**Pull request by rjl493456442**
This pull request pulls the latest changes from tests repo, including the changes for EIP7610 (https://github.com/ethereum/tests/pull/1367)

TODO 

- [x] "tests" are not fully consistent regarding the definition of the fork name. Both "Paris" and "Merge" are used in different places.

### CODE REVIEW

**Review by holiman — APPROVED** [review]
LGTM when green

File: tests/init.go [hunk 1]
```diff
@@ -246,7 +246,7 @@ var Forks = map[string]*params.ChainConfig{
 		ArrowGlacierBlock:   big.NewInt(0),
 		GrayGlacierBlock:    big.NewInt(0),
 	},
-	"Merge": {
+	"Paris": {
```
Comment (holiman) [inline tests/init.go:249]: If this causes a problem, we can just add `Merge` too, so we have them both

Comment (rjl493456442) [reply tests/init.go:249]: I have told testing guys to unify the test data. I am fine with defining both `Merge` and `Paris` to pass the tests.

Comment (rjl493456442) [reply tests/init.go:249]: Pushed

**Pull request merged**